robottelo/cli/import_.py

```diff
@@ -50,37 +86,65 @@ def activation_key(cls, options=None):
 
         """
         cls.command_sub = 'activation-key'
-        return cls.execute(
+        result = cls.execute(
             cls._construct_command(options),
             output_format='csv'
         )
+        if result.return_code == 0:
+            result.transition_data = cls.read_transition_csv(
```

Comment:
`result` is an instance of SSHCommandResult and by patching it we are breaking the meaning of that object is intended. Also this is breaking what is expected for every CLI helper class and method in the entire automation.

Even though this is very handy I don't agree on creating a special case within the CLI framework. To me makes more sense to create a specialized class that have method for retrieving and returning the trasition_data and that should be called on the test body and not inside a CLI helper.